common/changes/@gooddata/sdk-ui-all/ch4_2023-09-06-13-22.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@gooddata/sdk-ui-all",
+      "comment": "Sdk-backend-tiger support createdAt, createdBy, modifiedAt, modifedBy for insight, analytical dashboard, dashboard plugin, metric.",
```

Comment:
Sdk-backend-tiger now supports the following entity properties: 
- createdAt
- createdBy 
- modifiedAt
- modifedBy 
for the following analytical objects: 
- insight (visualization)
- dashboard 
- dashboard plugin 
- metric